libraries/ESP8266WiFi/src/ESP8266WiFiSTA.cpp

```diff
@@ -113,21 +113,22 @@ wl_status_t ESP8266WiFiSTAClass::begin(const char* ssid, const char *passphrase,
         return WL_CONNECT_FAILED;
     }
 
-    if(passphrase && strlen(passphrase) > 64) {
+    int passphrase_len = passphrase == NULL ? 0 : strlen(passphrase);

```

Comment:
Nitpick: Please use nullptr instead of NULL in C++ code

Reply:
Also, please prefer camelCase: passphrase_len => passphraseLen

Reply:
i've fixed.